### PR TITLE
Refactoring installation documentation

### PR DESCRIPTION
**Pull request by saadrahim**

### CODE REVIEW

File: docs/deploy/linux/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Deploy ROCm on Linux
 
 Please start with {doc}`/deploy/linux/quick_start` or follow the detailed
```
Comment (Rmalavally) [inline docs/deploy/linux/index.md:3]: Remove the word Please; it is never used in technical documentation.

File: docs/deploy/linux/install_overview.md [hunk 2]
```diff
@@ -1,78 +1,21 @@
...
-
-For the rest of the ROCm packages, you can find the licensing information at the
-following location: `/opt/rocm/share/doc/<component-name>/`
-
-For example, you can fetch the licensing information of the `_amd_comgr_`
-component (Code Object Manager) from the `amd_comgr` folder. A file named
-`LICENSE.txt` contains the license details at:
-`/opt/rocm-5.4.3/share/doc/amd_comgr/LICENSE.txt`
+Integrating with the distribution's package manager lets the user install,
+upgrade and uninstall using familiar commands and workflows. 3rd party ecosystem
```
Comment (Rmalavally) [inline docs/deploy/linux/install_overview.md:14]: Spell out Third party; never 3rd party

File: docs/deploy/linux/install_overview.md [hunk 3]
```diff
@@ -1,78 +1,21 @@
...
-- Allowing you to specify the use case and automating the installation of all
-  the required packages
-- Installing multiple ROCm releases simultaneously on a system
-- Automating updating local repository information through enhanced
-  functionality of the `amdgpu-install` script
-- Performing post-install checks to verify whether the installation was
-  completed successfully
-- Upgrading the installed ROCm release
-- Uninstalling the installed single-version or multi-version ROCm releases
+The `amdgpu-install` is a wrapper around the package manager. The same packages
```
Comment (Rmalavally) [inline docs/deploy/linux/install_overview.md:17]: Remove The

File: docs/deploy/linux/install_overview.md [hunk 4]
```diff
@@ -83,29 +26,11 @@ case with most convenience wrappers, some degree of customization is lost for
 the sake of simplicity.
 ```
```
Comment (Rmalavally) [inline docs/deploy/linux/install_overview.md:27]: This tip is vague. Provide more details.

File: docs/deploy/linux/installer/index.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# AMDGPU Install Script
+
+Please follow these steps to use
```
Comment (Rmalavally) [inline docs/deploy/linux/installer/index.md:3]: Remove 'Please'

File: docs/deploy/linux/installer/index.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+:link-type: doc
+
+Steps for removing ROCm packages libraries and tools.
+:::
+
+::::
+
+## See Also
+
+- {doc}`/release/gpu_os_support`
```
Comment (Rmalavally) [inline docs/deploy/linux/installer/index.md]: We could consider adding "how to uninstall ROCm" here.

File: docs/deploy/linux/installer/upgrade.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# Upgrade via Installer Script (Linux)
+
```
Comment (Rmalavally) [inline docs/deploy/linux/installer/upgrade.md:2]: Using Installer Script for Upgrades

File: docs/deploy/linux/installer/index.md (hunk 5 above)
Comment (samjwu) [reply docs/deploy/linux/installer/index.md]: [update AMDGPU Install Script page](https://github.com/RadeonOpenCompute/ROCm/pull/2202/commits/04f65933072e38f7569b32ce90c7abed7529441b)

**Review by samjwu — APPROVED** [review]
Approving this now to include important fixes.

Another round of fixes to follow.

**Pull request merged**